src/widgets/avformatproducerwidget.ui

```diff
@@ -841,7 +867,7 @@
        </property>
        <property name="icon">
         <iconset theme="show-menu" resource="../../icons/resources.qrc">
-         <normaloff>:/icons/oxygen/32x32/actions/show-menu.png</normaloff>:/icons/oxygen/32x32/actions/show-menufill.png</iconset>
+         <normaloff>:/icons/oxygen/32x32/actions/show-menu.png</normaloff>:/icons/oxygen/32x32/actions/show-menufull.png</iconset>
```

Comment:
This is a weird change, but as far as I can tell from looking at other UI XML files, there should be neither a "fill" or a "full" here.

Reply:
Yes. Odd. Qt Creator does strange things sometimes. I do not like that it doesn't keep gridview items in order in the XML.

Fixed.